_data/zhoukenny/zhoukenny.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Kenny Zhou
+institution: Cornell University 🚩
+profile_pic: Zhou_Kenny-Headshot.jpg The picture must be squared and 544px on width and height.
```

Comment:
Please do one of 2 things:
* Add back the hash symbol `#` before the comment
Before: `Zhou_Kenny-Headshot.jpg The picture must be squared and 544px on width and height.`
After:`Zhou_Kenny-Headshot.jpg # The picture must be squared and 544px on width and height.`
* Remove the comment
Before: `Zhou_Kenny-Headshot.jpg The picture must be squared and 544px on width and height.`
After: `Zhou_Kenny-Headshot.jpg`